packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1006,6 +1006,24 @@ class RawScrollbar extends StatefulWidget {
        assert(minOverscrollLength == null || minOverscrollLength >= 0),
        assert(radius == null || shape == null);
 
+  /// Creates a set of scrollbars for vertical and horizontal axes.
+  factory RawScrollbar.dual({
+    required ScrollController verticalController,
+    required ScrollController horizontalController,
+    required Widget child,
+  }) {
+    // TODO(Piinks): Instead, `.dual` should be a constructor that internally
+    //  creates two ScrollbarPainters so that they can respond to each other
+    //  appropriately: https://github.com/flutter/flutter/issues/122345
+    return RawScrollbar(
```

Comment:
@TahaTesser @xu-baolin I wonder if you have any thoughts here.
There are a lot of properties in the RawScrollbar constructor that we would want to expose here for the dual constructor, but I do not want to enumerate them all twice (vertical ones and horizontal ones)

WDYT of a ScrollbarDetails or ScrollbarStyle class that can encapsulate all of the styling parameters and allowing folks to pass one for vertical and one for horizontal?
Alternatively, we could leave this for now in its simplest form and folks can override this?

Not sure. We do not have to do it all in one change either. But I am curious if you have thoughts. :)

Reply:
Another point to consider, there may be other unique ways users want to configure dual scrollbars. I filed https://github.com/flutter/flutter/issues/122351 for example as one thing I thought of.

Reply:
I think we should expose basic properties to satisfy the common usage.
Also, let it be able to do custom easily.

Reply:
I filed these for follow up, there is definitely going to be multiple changes to get to all of the features needed to do this right :)

- https://github.com/flutter/flutter/issues/122351
- https://github.com/flutter/flutter/issues/122345